test/multisigwallet/is_transaction_confirmed.js

```diff
@@ -31,8 +31,7 @@ contract('MultiSigWallet::isTransactionConfirmed', async (accounts) => {
                 nonExistingTransactionID,
                 { from: helper.wallet(0) },
             ),
-            'Should revert as transaction ID does not exist.',
-            'Transaction does not exist.',
```

Comment:
Why this check has been removed?

Reply:
running truffle test gave me below error:

 AssertionError [ERR_ASSERTION]: 
The contract should revert with:
	"Transaction does not exist." 
instead received:
	"Returned error: VM Exception while processing transaction: revert"